### DISCUSSION TITLE
Do you use docs inside of the Directus Data Studio?

### DISCUSSION THREAD

**Discussion by phazonoverload** [directus/directus#16905] [Archive]
Hey hey,

I've recently started as the [Developer Relations Lead at Directus](https://directus.io/blog/developer-relations-first-month/) and got a quick question for you. 

Do you ever use the docs that we provide in the Directus Data Studio? Or do you use them standalone at [docs.directus.io](https://docs.directus.io)?

<img width="1435" alt="Screenshot of Data Studio with the docs visible, with a big red arrow to a question mark icon in the sidebar." src="https://user-images.githubusercontent.com/1461554/209414512-ce30ee02-9ffb-4b97-8a2b-98fbf306d6df.png">

We're making lots of docs improvements, including stuff around how we present information, and injecting changes into the app involves some engineering overhead. Thought this would be a good point to just ask what people think about this feature.

Any feedback welcome! ❤️

**Comment by yisheng** [comment]
- Docs in the Directus Data Studio: 0%
- [docs.directus.io](https://docs.directus.io/): 100%

**Reply by sptv2reg** [reply]
Agree. Never use it. The main reason is that I always google first and then proceed to docs.directus.io

**Reply by raphourbe** [reply]
Same here, I never use this doc. I always go to the main one.

**Comment by ernst86913** [comment]
Never use it for this purpose - always docs.directus.io. But it would be great if this module could be adapted to a custom application documentation… eg one codes a e-commerce marketplace… and the documentation can be written there using MD files which are showing in this module…

**Comment by paescuj** [comment]
To be honest, I always use [docs.directus.io](https://docs.directus.io), but I can imagine that the inbuilt docs are helpful for end-users.

**Comment by Pachat** [comment]
As others have said, I never use it and even disabled it in project settings.
First I use Google, then I go to the very useful [docs.directus.io](https://docs.directus.io/) which helps with explanations and examples for developers.
For end users, I have a custom link in the mod bar that points to a custom documentation for the app.

As this is your first post, let me say "Welcome aboard"
Very happy to see the care Directus has for the developers.

**Comment by phazonoverload** [comment]
Really super appreciate the feedback everyone. This kind of confirms an assumption we've had but it's great to not just assume. Happy new year!
